src/test/java/com/java8/wordsplay/FileReaderTest.java

```diff
@@ -83,7 +83,7 @@ void shouldReturnWordWithSortedEntries() throws Exception {
 //        - row​: 2
 //                - car​: 1
 //                - die​:
-       String words =  "ant boat row ant car ant die ant boat row ant"  ;
+       String words =  "zan zan zan zan zan zan zan zan ant boat row ant car ant die ant boat row ant zan zan"  ;
 
         String[] wordsArray = words.split(" ");
         List<String> wordList = Arrays.asList(wordsArray);
```

Comment:
objects.nonNull check here before we iterate on the list.

Reply:
Good one will fix